subscriptions-api.md

```diff
@@ -0,0 +1,613 @@
+# CNCF CloudEvents – Subscriptions API
```

Comment:
just for consistency with the other spec perhaps we should add ` - Version 0.1-rc01`